### PR TITLE
docs: Adding return vue instance in mount method(ecosystem-vue.md)

### PR DESCRIPTION
**Pull request by kwen8**
update docs

### CODE REVIEW

File: docs/ecosystem-vue.md [hunk 1]
```diff
@@ -7,45 +7,51 @@ sidebar_label: Vue
...
 The CLI Plugin does the following for you:
-1) Modify your webpack config so that your project works as a single-spa application or parcel.
-2) Install [single-spa-vue](https://github.com/CanopyTax/single-spa-vue).
-3) Modify your `main.js` or `main.ts` file so that your project works as a single-spa application or parcel.
-4) Add a `set-public-path.js` that will use `systemjs-webpack-interop` in order to set the public path of your application.
+
+1. Modify your webpack config so that your project works as a single-spa application or parcel.
+2. Install [single-spa-vue](https://github.com/CanopyTax/single-spa-vue).
+3. Modify your `main.js` or `main.ts` file so that your project works as a single-spa application or parcel.
+4. Add a `set-public-path.js` that will use `systemjs-webpack-interop` in order to set the public path of your application.
```
Comment (frehner) [inline docs/ecosystem-vue.md:32]: can we change these to just `1.` for all of them? that way we don't need to keep the list ordered in future revisions. :)

**Review by frehner — COMMENTED** [review]
small comment, otherwise looks good. thanks!

File: docs/ecosystem-vue.md (hunk 1 above)
Comment (kwen8) [reply docs/ecosystem-vue.md:32]: Sure, I think it's ok

**Review by internettrans — APPROVED** [review]
I think it's fine without changing all to 1's. Better to merge this in than let it sit waiting on that I think

**Pull request merged**